src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.alerts
```

Comment:
added this to `alerts` package rather than `wiremock` - I thinking it is better to keep this together (maybe a sub wiremock package - not sure)